docs/kyma/docs/029-details-install-subcomponents.md

```diff
@@ -43,4 +43,24 @@ data:
   azure-broker.enabled: "true"
 ```
 
->**NOTE:** Some subcomponents can require additional configuration to work properly.
\ No newline at end of file
+>**NOTE:** Some subcomponents can require additional configuration to work properly.
+
+## Specify subcomponents versions
+
+Versions of the Kyma components are specified in `values.yaml` file in charts. Each component version is described by two properties: `version` and `dir`. The first defines actual docker image tag. The second describes directory under which the tagged image is pushed follwed by a `/` or this value can be empty.
```

Comment:
Pls change into (without blockquotes ;) :
 > Versions of the Kyma components are specified in the `values.yaml` file in charts. Two properties, `version` and `dir`, describe each component version. The first one defines the actual docker image tag. The second property describes the directory under which the tagged image is pushed. It is optional and  is followed by a forward slash (/).